### PR TITLE
feat(resource-status): Implement services tab

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This adds a services tab for resources of type host. All services are clickable (through their name), and redirects to the corresponding details

Peek 2020-09-16 11-33

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 20.10.x (master)

### CODE REVIEW

File: www/front_src/src/Resources/Details/tabs/Services/index.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      <Skeleton height={25} />
+      <Skeleton width={50} height={25} />
+    </div>
+  );
+
+  return (
+    <div className={classes.services}>
+      {serviceLoadingSkeleton}
+      {serviceLoadingSkeleton}
+      {serviceLoadingSkeleton}
```
Comment (jdelpierre) [inline www/front_src/src/Resources/Details/tabs/Services/index.tsx:51]: Why 3 same occurences ?

**Review by jdelpierre — DISMISSED** [review]

File: www/front_src/src/Resources/Details/tabs/Services/index.tsx (hunk 1 above)
Comment (bdauria) [reply www/front_src/src/Resources/Details/tabs/Services/index.tsx:51]: these are loading skeletons, we could display 2 of them or even 4, I just thought 3 might be enough

**Comment by bdauria** [comment]
> As `duration` could be null, isn't-it better to type `duration` as `string | null` ? Thus decode `duration` with `JsonDecoder.nullable(JsonDecoder.string)`

done

**Comment by Thebarda** [comment]
LGTM with latest changes ✔️

**Review by Thebarda — DISMISSED** [review]

**Review by kduret — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**